lib/eye/notify/awssdk.rb

```diff
@@ -0,0 +1,43 @@
+require "aws-sdk-core"
+require "aws-sdk-core/ses"
+
+module Eye
+  module Notify
+    class AWSSDK < Eye::Notify
+      param :region, String, true
+      param :access_key_id, String, true
+      param :secret_access_key, String, true
+      param :from, String, true
+
+      def execute
+        client = Aws::SES::Client.new(
+          region: region,
+          credentials: Aws::Credentials.new(access_key_id, secret_access_key))
```

Comment:
http://docs.aws.amazon.com/AWSSdkDocsRuby/latest//DeveloperGuide/prog-basics-creds.html#creds-default

Aws is configurable via the default provider so that the client code itself never has to specify credentials. I think a lot of people use this setup, so we shouldn't force people to configure credentials in the yaml/erb .